02-basic-programming/04-final-products/05-solutions-final-products/README.md

```diff
@@ -1,4 +1,4 @@
-# Produtos Finais Solucionados
+# Solução dos Produtos Finais
```

Comment:
As the soft-skills parts were removed, we should rename this folder in order to follow the enumeration right.